README.md

```diff
@@ -129,6 +131,19 @@ Solomon proposed to support VXLAN from the Docker core.
 - Docker Hub Enterprise
 - Private Docker registries using the “registry" docker image.
 
+## Configuration Management Tools with Docker support 
+
+- Ansible 
+- Saltstack
+- Chef 
+- Puppet 
+- Packer
```

Comment:
I agree on Packer because of https://packer.io/docs/builders/docker.html but, IMHO the other have nothing to do with Docker. They can be used inside docker, like bash, but it doesn't specifically target docker!